dll/win32/shell32/precomp.h

```diff
@@ -36,6 +36,7 @@
 #include <powrprof.h>
 #include <winnetwk.h>
 #include <objsafe.h>
+#include <regstr.h>
```

Comment:
Are the functions from this header used in more than one file?

Reply:
No functions, its the REGSTR_PATH_EXPLORER string. And I'd love for other files to use this instead of the current inconsistency of "Software\..." vs "SOFTWARE\...".